docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-solution:
+
+Relying Party Solution
+++++++++++++++++++++++
+
+This document defines the implementation of the online presentation and verification flow of a credential (PID or (Q)EAA), in accordance with the "OpenID for Verifiable Credential Presentation - draft 19" specifications [OID4VCP], in the context of a Cross-Device flow scenario.
+
+
+Cross Device Authentication
+---------------------------
+
+In the Cross Device Authentication, the user logs into the website and provides their consent for presenting their PID through the Wallet. 
+Once authentication is performed, the end user can request the display of attributes provided by the Wallet.
+Below is explained the Cross Device Authentication flow.
+
+NOTE: Use Cases about other flows will be analysed in next sprints.
+
+
+.. image:: ../../images/cross_device_auth_seq_diagram.svg
+  :align: center
+  :target: http://www.plantuml.com/plantuml/umla/TLFDZjem4BxxAKPKgjf3Ns0bgkrIMj5M8eKiU-7Y91DmrnbJ_soMFdtjiCOcY5iqy_rydntV6ykrTPrat5b5hgjGCtRmCr6B0oSBaqU3UWBSW6EiKgymMQ4y2jf1uL772Rpx9NPx-o0TNl8sg4KhKCE7RrgHdLFpSP1fR-8UUFryXbN8a1hmZgCyJrnAjB0W7vrg7C0zOzCfV75sZ-IHt0f50DCfS_3fC_HtmqffyG-X5tOF9mt6QojUk4LmwRDdVU2qU0VhS3PdwY3A6ap8H6gHjHXebQVDD8RP1GzM-DUXIPQXN-Kf9wkuXiVL8hUeecwRT7-lOAQQkEXzpBtg_JGCyngiZ_kQqvaLX_DNgpquDzvIgrMNaB7FztbvXYshF-XPMwgEEVMwLZ0PiKR5Of8Dbw89inzF9Qp5ZhXrEgqBhMeqPfpW_PQowqO8VscAN2pNvTK5c8CYWwCR7EKEz9kH4Y26kccHwxATLku0XP8oF1ld8qlG4TjYEkTHvk6KIyt913b5YwytyiwyakiGDWMKwXafMote9PR9bAwLyvzn-NFK0AbXRq5TzfQQ7DPUr7RitfDS9_y3
+
+.. list-table:: 
+  :widths: 25 25 25 25
+  :header-rows: 1
+
+  * - **Id**
+    - **Source**
+    - **Destination**
+    - **Description**
+  * - **1**
+    - **User**
+    - **Wallet Instance**
+    - The User navigates the Verifier's website and clicks on the 'Login with IT Wallet' button
+  * - **2**
+    - **User**
+    - **Wallet Instance**
+    - The Authorization flow starts
+  * - **3-4**
+    - **Verifier (FE)**
+    - **Verifier (BE)**     
+    - Requests the generation of an Authorization Request which contains the Presentation Definition/Scope, Verifier metadata and information the wallet needs to generate the response.
+  * - **5**
+    - **Verifier (FE)**
+    - **-**     
+    - Inserts the reference to the previously generated object into a QR Code that is displayed on the screen
+  * - **6**
+    - **Verifier (FE)**
+    - **Wallet**
+    - QR Code is displayed ready to be scanned
+  * - **7**
+    - **Wallet**
+    - **-**
+    - Scans the QR Code
+  * - **8**
+    - **Wallet**
+    - **-**
+    - Extracts the Request URI from the payload of the QR Code
+  * - **9-10**
+    - **Wallet**
+    - **Verifier (BE)**          
+    - Requests the content of the Authorization Request by invoking the Request URI
+  * - **11**
+    - **Wallet**
+    - **-**     
+    - Verify the signature of the Request Object (JWT)
```

Comment:
between 11 and 12, please add the user interaction in consent management and selective disclosure